src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -675,7 +675,7 @@ case object CREATE extends OpCode(0xf0, 3, 1, _.G_create) {
     } else {
 
       val (initCode, memory1) = state.memory.load(inOffset, inSize)
-      val (newAddress, world1) = state.world.newAddress(state.env.ownerAddr)
+      val (newAddress, world1) = state.world.increaseCreatorNonceAndCreateAddress(state.env.ownerAddr)
```

Comment:
@ntallar it is necessary to perform this operations always together (I mean increase creator nonce and create address) if yes then maybe only name is misleading?